src/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "outDir": "../out",
     "allowJs": false,
     "sourceMap": true,
-    "target": "ES2016",
+    "target": "esnext",
```

Comment:
it seems also need add `libs: ["esnext", "esnext.string"]`. But better try locally first via `npm run check` and `npm run build`. If it still fail just revert `@ts-ignore`

Reply:
That worked, thanks!